tests/functional/invalid-encoding/source/data/config.json

```diff
@@ -0,0 +1,5 @@
+{
+	"parameters": {

```

Comment:
tohle šlo otestovat unittestem na Config v izolaci

Reply:
Ha, koukám, že dole to je otestované. Tak proč ještě funkcionálně?

Reply:
protoze chci vedet, ze se to konvertuje na spravnou chybu, kvuli tomu error handleru